cms/djangoapps/contentstore/views/item.py

```diff
@@ -273,6 +273,17 @@ def xblock_view_handler(request, usage_key_string, view_name):
                     content_type="text/plain",
                 )
 
+            try:
```

Comment:
In general, check your code coverage report. I know that some of that code is tested by bok choy tests, but that generally only covers "happy path" scenarios. And where unit testing can be easily added, there is no reason not to do both.